ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -50,6 +50,27 @@ FCU_DFSS_CSP_DXZC:
   - CSP
   - DXZC
 
+FCU_DXRC_RMM:
```

Comment:
Where is the information for the discharge fan? should have `DFSS` at least.

Reply:
even if incomplete in the translation.

Reply:
Updated